trinity.html

```diff
@@ -0,0 +1,1172 @@
+<head>
+  <link rel="stylesheet" type="text/css" href="trinity.css">
```

Comment:
This is here so i could test it out in my local browser, this shouldn't be here long term.